src/message/messageActionSheet.js

```diff
@@ -12,7 +12,7 @@ import { navigateToMessageReactionScreen } from '../nav/navActions';
 // TODO really this belongs in a libdef.
 export type ShowActionSheetWithOptions = (
   { options: string[], cancelButtonIndex: number },
-  (number) => void,
+  (number) => Promise<void>,
```

Comment:
This isn't our type to modify, unfortunately – as the comment notes, it shouldn't really be here at all. (Also, this modification results in a Flow error.)

On the other hand, `ButtonDescription` (seen below) *is* ours.

Reply:
Fixed